onnxruntime/core/framework/op_kernel.cc

```diff
@@ -218,4 +221,147 @@ Status OpKernelContext::SetOutputMLValue(int index, const OrtValue& ort_value) {
 }
 #endif
 
+// InstantKernelContext
+InstantKernelContext::InstantKernelContext(const OrtValue* const* input_values,
```

Comment:
So `InstanceKernelContext` is the "minimalistic" OpKernelContext required by an ORT kernel to execute, is that correct ?

Reply:
yes we could deem it that way - yet it is not a real "subset" of "OpKernelContext".